examples/daal/cpp_sycl/source/logistic_regression/log_reg_dense_online.cpp

```diff
@@ -0,0 +1,143 @@
+/* file: log_reg_dense_online.cpp */
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+!  Content:
+!    C++ example of logistic regression in the online processing mode
+!    with DPC++ interfaces.
+!
+!    The program trains the logistic regression model on a training
+!    datasetFileName and computes classification for the test data.
+!******************************************************************************/
+
+/**
+ * <a name="DAAL-EXAMPLE-CPP-LOG_REG_DENSE_ONLINE"></a>
+ * \example log_reg_dense_online.cpp
+ */
+
+#include "logistic_regression_online.h"
+#include "service.h"
+#include "service_sycl.h"
+
+using namespace daal;
+using namespace daal::algorithms;
+using namespace daal::algorithms::optimization_solver;
+using namespace daal::data_management;
+
+/* Input data set parameters */
+const std::string trainDatasetFileName = "../data/batch/logreg_train.csv";
+const std::string testDatasetFileName  = "../data/batch/logreg_test.csv";
+constexpr size_t nFeatures             = 6;
+constexpr size_t nClasses              = 5;
+
+logistic_regression::ModelPtr trainModel();
+void testModel(const logistic_regression::ModelPtr& model);
+
+int main(int argc, char * argv[])
+{
+    checkArguments(argc, argv, 2, &trainDatasetFileName, &testDatasetFileName);
+
+    for (const auto & deviceSelector : getListOfDevices())
+    {
+        const auto & nameDevice = deviceSelector.first;
+        const auto & device     = deviceSelector.second;
+        cl::sycl::queue queue(device);
+        std::cout << "Running on " << nameDevice << "\n\n";
+
+        services::SyclExecutionContext ctx(queue);
+        services::Environment::getInstance()->setDefaultExecutionContext(ctx);
+
+        auto model = trainModel();
+        printNumericTable(model->getBeta(), "Logistic Regression coefficients:");
+
+        testModel(model);
+    }
+
+    return 0;
+}
+
+logistic_regression::ModelPtr trainModel()
+{
+    constexpr bool isIntercept           = false;
```

Comment:
[style] Cool of course, but in other examples, no one does that. Looks probably not consistency.

Reply:
Where do you see the inconsistency?